eng/scripts/Export-API.ps1

```diff
@@ -3,9 +3,25 @@ param (
     [Parameter(Position=0)]
     [ValidateNotNullOrEmpty()]
     [string] $ServiceDirectory,
-    [string] $SDKType = "all"
+    [string] $SDKType = "all", 
+    [switch] $SpellCheckPublicApiSurface
 )
 
+if ($SpellCheckPublicApiSurface -and (Get-Command npx | Measure-Object).Count -eq 0) { 
```

Comment:
PowerShell uses falsy conditions like JS, so this doesn't need to be so complicated:
```suggestion
if ($SpellCheckPublicApiSurface -and (Get-Command npx)) { 
```